finance-api/api-reference.md

```diff
@@ -0,0 +1,282 @@
+
+```
+GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/balance
+```
+
+See example [Fetch balance for a liquid account](#fetch-balance-for-a-liquid-account).
+
+### Parameters
+
+<details open="true">
+<summary>Parameters</summary>
+
+|Name |Type |In |Required/Optional |Description
+|:---- |:---- |:---- |:---- |:----
+|organizationUuid |string |path |required |Unique identifier for your organization. You can specify the value with one of the following: <br/><ul><li> Use `self` as the value. This will retrieve your organizationUuid from the authentication token in the request.</li><li> Get it by using the https://oauth.izettle.com/users/me endpoint of OAuth2 API. See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information.</li></ul> 
+|accountTypeGroup |string |path |required |The account type from which the data like transactions is retrieved. You can use one of the following account types: <br/><ul><li> `PRELIMINARY` account where transactions are to be confirmed by third-party acquirers.</li><li> `LIQUID` account where transactions are to be paid out to the merchant.</li></ul>
+|at |string |query |optional |Used to fetch account balance that is available at a time in the UTC time zone. If it's used, any transaction after that point will be ignored. If it's not used, the balance of all transactions at the current point of time is returned. <br/>You can specify a UTC time in one of the following formats: <br/><ul><li>`YYYY-MM-DD` to specify a date. For example, `2020-11-29`.</li><li>`YYYY-MM-DDThh:mm:ss` to specify a time. For example, `2020-11-29T03:10:02`.</li></ul>
+</details>
+
+
+### Responses
+<details open="true">
+<summary>HTTP status codes</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the account balance is successfully fetched.  
+|403 Forbidden |Returned when you are not authorised to fetch the account balance.
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request .  
+</details>
+
+<details open="true">
+<summary>Response attributes</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |object|Information about the account balance at a point in time. It's represented by `totalBalance` and `currencyId`.
+|totalBalance |integer |The account balance. It can be negative. For example, `-300`
+|currencyId |string |The currency of the account balance. For example, `SEK`.
+</details>
+
+
+## Fetch account transactions
+Returns all transactions or transactions of certain types from a merchant's Zettle preliminary or liquid account during a specific period.
+
+```
+GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/transactions?{start}&{end}
```

Comment:
Curly braces are usually used to express interpolation of a value.
Putting it like this "{start}&{end}" doesn't seem to be right since every query parameter has a name.
```suggestion
GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/transactions?start={start_date}&end={end_date}
```

This needs to be adjusted in many places.